### DISCUSSION TITLE
Wikilink Formats

### DISCUSSION THREAD

**Discussion by anaclumos** [notaapp/talk#314] [Questions] [answered]
Currently, if I type `[[` to make a custom link, it will paste the wikilink as `FolderName/DocName`. Is it possible to modify this, for example, to only paste the `DocName`?

**Comment by anaclumos** [comment]
I also figured that this happens to pasted images too. Given that pasted images (assets) are unique, can we link them as `![[ImageName.png]]`, not `![[FolderName/ImageName.png]]`?

**Comment by astoilkov** [comment, marked as answer]
Hmm. That's strange. By default this option is disabled. You can toggle it from the Command Palette (⌘⇧P):
![image](https://user-images.githubusercontent.com/884810/193280377-af8a030f-4767-4d82-93d8-c0e9e5e91a0f.png)

Do you know if you have accidentally selected this or do you think it was like that when you installed Nota?

**Reply by anaclumos** [reply]
This was it! Thank you. One rec -- I guess this option should be inside Settings?

**Reply by astoilkov** [reply]
Yeah, we should move it.
